osm_fudge/metrics/levenshtein.py

```diff
@@ -8,7 +8,7 @@
 import numpy as np
 import argparse
 
-def find_levenshtein_distance(str1, str2):
+def find_levenshtein_distance(str1, str2, options):
```

Comment:
Ah, now I see. We need the dummy options here for the `n: 2` object we pass in for bi-grams. Hm, maybe we can refactor this?

Reply:
Do you mean removing the options here and make it compulsary only for jaccard function? In that case, we would bind options only with jaccard distance metrics.